internal/integration_test/bench/bench_test.go

```diff
@@ -89,9 +89,10 @@ func runCompilation(b *testing.B, r wazero.Runtime) wazero.CompiledModule {
 func runInitializationBench(b *testing.B, r wazero.Runtime) {
 	compiled := runCompilation(b, r)
 	defer compiled.Close(testCtx)
+	config := wazero.NewModuleConfig().WithSysNanotime().WithSysWalltime().WithRandSource(rand.Reader)
```

Comment:
same comment as in #842

Reply:
responded in #842. It's the same commit so whatever resolution you prefer will apply to both.

Reply:
I think a code-comment will work!